File: 1-js/1-js-basics/5.1-while-loop-lab/05-accountBalance.js
function solve(arr) {

    let index = 0;

    let current = arr[index];

    let result = [];

    let sum = 0;

    while (current != 'NoMoreMoney') {

        current = Number(current);

        if (current < 0) {

            result.push('Invalid operation!');

            break;

        }

        sum += current;

        result.push(`Increase: ${current.toFixed(2)}`);

        index++;

        current = arr[index];

    }

    result.push(`Total: ${sum.toFixed(2)}`);

    return result.join('\n');

}

console.log(solve(["120",
    "45.55",
    "-150"])
);

File: 2-python/1-python-basics/5.1-while-loop-lab/05-account_balance.py
deposit = input()
money = 0

while deposit != 'NoMoreMoney':
    deposit = float(deposit)
    if deposit < 0:
        print('Invalid operation!')
        break
    else:
        print(f'Increase: {deposit:.2f}')
        money += deposit
        deposit = input()

print(f'Total: {money:.2f}')
